packages/blitz/scripts/postinstall.js

```diff
@@ -21,6 +22,45 @@ try {
   // noop
 }
 
+async function findNodeModulesRoot(src) {
```

Comment:
So that's not perfect, because I copy pasted this function from `routes.ts` with some changes like: removed types, used different imports, `isInBlitzMonorepo`. Open for suggestions how to make it better.